php-nginx/Dockerfile

```diff
@@ -16,9 +16,6 @@
 
 FROM gcr.io/google_appengine/debian8
 
-# Cache busting for security update
-ENV UPDATED_AT 2016-11-15
```

Comment:
Do we no longer want to keep this? It still feels useful to have an easy mechanism to bust the docker cache. Maybe `UPDATED_AT` is a misnomer as it may not reflect the last time this file or image was updated.